### PR TITLE
Added the form with php code to submit.php

### PR DESCRIPTION
**Pull request by GizKockesen**
So I've been going through tutorials and I think this works (hopefully).
I added the php code that should be able to add the necessary info
collected from the form to the Idea table in the database. Since I don't
have the info for connecting to the database, I didn't add the database
connection lines. I really hope this works!

### CODE REVIEW

**Comment by alex9311** [comment]
yeah that would be great if you want to add it! 

I just did a pull request to merge my upload image example to your branch here: https://github.com/alex9311/TUD-Information-Retrieval-Group-02/pull/44

I thought it would be cool to get the full submission working on your branch then merge that into master once its done. What do you think?

**Comment by GizKockesen** [comment]
I agree, on it! :)

**Comment by alex9311** [comment]
Great, let me know when when you're done! Then I can try to make it so we're storing the image's location on the server in the database as the image field.

**Comment by GizKockesen** [comment]
Are we having two forms then?

**Comment by GizKockesen** [comment]
Like nested, I mean

File: app/pages/file_upload.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    echo "Sorry, only JPG, JPEG, PNG & GIF files are allowed.";
+    $uploadOk = 0;
+}
+// Check if $uploadOk is set to 0 by an error
+if ($uploadOk == 0) {
+    echo "Sorry, your file was not uploaded.";
+// if everything is ok, try to upload file
+} else {
+    if (move_uploaded_file($_FILES["fileToUpload"]["tmp_name"], $target_file)) {
+	session_start();
```
Comment (alex9311) [inline app/pages/file_upload.php]: This is where we want to add the database entry stuff

**Comment by alex9311** [comment]
Ah yeah, I was thinking about it wrong actually.

See how my image uploader calls app/pages/file_upload.php when you press submit? That's submitting through a different file, like we were talking about before.

We want a single form, so only having to press submit once. It might be easiest to move your database login into app/pages/file_upload.php. Then we can upload the file and then make the entry into the database. That way we can get the file location from the file uploading logic (the $target_file variable) and store that into the database. 

Does that make sense? I made a comment in the code where I'm thinking the database stuff should go, if you look [here](https://github.com/alex9311/TUD-Information-Retrieval-Group-02/pull/43/files)

**Comment by GizKockesen** [comment]
Should I move the database entry lines for text_description to file_upload.
php as well then? And also change the form action to file_upload. php?

**Comment by alex9311** [comment]
yeah that should work!

**Comment by GizKockesen** [comment]
Okay I updated the two files (submit and file_upload). Let me know if I
misunderstood something :)

**Comment by alex9311** [comment]
Thanks! Code looks good, I think you just had one small mistake. I made the fix in a separate branch and made a pull request to bring it into yours here: https://github.com/alex9311/TUD-Information-Retrieval-Group-02/pull/45

I also restructured the code a bit in file_upload.php because it was getting a bit cluttered.

Let me know what you think! I tested it and its working!

**Comment by GizKockesen** [comment]
It's great! And yaayyy it's working :D

**Comment by alex9311** [comment]
Great! Should we merge this into master now?

I could try and quickly make a way to view what is in the database

**Comment by GizKockesen** [comment]
Yeah sure!

**Comment by alex9311** [comment]
I added a view_ideas.php page which prints the ideas in the database with images. I'm going to merge this branch in now!

**Comment by GizKockesen** [comment]
awesome!

**Pull request merged**